request.go

```diff
@@ -104,6 +118,27 @@ func (client *Client) doJsonRequest(method, api string,
 	return nil
 }
 
+func (client *Client) getRateLimit(response *http.Response) error {
+
+	ratelimit := response.Header.Get(RateLimitHeader)
+	if ratelimit == "" {
```

Comment:
It doesn't supper matter the header.I just wanna verify one of them is there and then create the rate limit object to return back.